features/bsm/rest/api/src/main/java/org/opennms/web/rest/v2/bsm/model/edge/AbstractEdgeResponseDTO.java

```diff
@@ -46,6 +46,7 @@
 import org.opennms.web.rest.api.support.JsonResourceLocationSerializationProvider;
 import org.opennms.web.rest.v2.bsm.model.MapFunctionDTO;
 
+import com.google.common.base.MoreObjects;
 import com.google.common.base.Objects;
```

Comment:
Remove this import, too (and replace with `MoreObjects` or the jdk `Objects` alternatives.